### PR TITLE
Joining Feat/categories to branch Dev

### PR DESCRIPTION
**Pull request by sZeuSz**
Here i have implemented the backend of home page

Feature covered:

 Tests with jest;
 Page home backend;

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import express from 'express';
 import cors from 'cors';
+import signup from './controllers/signup.js';
```
Comment (calilrenner) [inline src/app.js]: Hey Roseno, you can delete signup testing before we merge with dev. I am currently working on signup and many things has changed. I also see some duplicate files, like 'mock2', is it right? Maybe it is my computer.

File: src/app.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+import categories from './controllers/categories.js';
 import signin from './controllers/signin.js';
 
 const app = express();
 
 app.use(express.json());
 app.use(cors());
 
+app.post('/register', signup);
```
Comment (calilrenner) [inline src/app.js]: you can delete this as well.

**Review by calilrenner — COMMENTED** [review]
Well done. Here goes some tips. When we were at classroom, Leandro told that it is better to test exceptions. So, I think that when analyzing body response, we can see if the response cames with a massage object or if the response has same properties like object keys. But do not worry, we can move on, since it does what it says.
Please, could you answer my doubts in the code comments? After that, I will merge. Thanks!

File: src/app.js (hunk 1 above)
Comment (sZeuSz) [reply src/app.js]: Solved.

File: src/app.js (hunk 2 above)
Comment (sZeuSz) [reply src/app.js]: Solved.

**Review by calilrenner — APPROVED** [review]
Good work! @sZeuSz

**Pull request merged**